suprb/optimizer/rule/es/es.py

```diff
@@ -16,6 +16,29 @@
 from ..origin import Matching, RuleOriginGeneration
 
 
+class AdaptMutation:
```

Comment:
wouldn't this be more appropriate in mutation.py?

Reply:
Yes. Moved it there